api/envoy/config/route/v3/route_components.proto

```diff
@@ -968,7 +968,9 @@ message RouteAction {
   // limits. By default, if the route configured rate limits, the virtual host
   // :ref:`rate_limits <envoy_api_field_config.route.v3.VirtualHost.rate_limits>` are not applied to the
   // request.
-  google.protobuf.BoolValue include_vh_rate_limits = 14;
+  //
+  // This field is deprecated. Please use :ref:`vh_rate_limits <envoy_v3_api_field_extensions.filters.http.ratelimit.v3.RateLimitPerRoute.vh_rate_limits>`
+  google.protobuf.BoolValue include_vh_rate_limits = 14 [deprecated = true];
```

Comment:
Since you are already doing all of this work, are you willing to deprecate `rate_limits` above also? Optimally we would remove all rate limit specific fields from this generic proto as part of this work? Feel free to say no but it would be highly appreciated and related to this work. :)

Reply:
Sure! I can do it as separate pull request so to not block this change from landing

Reply:
OK sounds good if you are willing to do this as a follow up. I will review this one.